### ISSUE TITLE
Problem install Chrome and Firefox in Raspberry

### ISSUE THREAD

**Issue by gustafsom** [InstaPy/InstaPy#2147]
Hi everyone, I'm trying to run Instapy on Raspberry pi 3. But there's an error with chrome: `Message: 'chromedriver' executable may have wrong permission.`

As a solution I tried to run through Firefox, but the error occurs: `Message: 'geckodriver' executes needs to be in PATH`

Please, I need help with this.

**Comment by KShimanski** [comment]
@gustafsom download the proper [geckodriver](https://github.com/mozilla/geckodriver/releases) depending upon your operating system, copy/move the driver to /usr/local/bin and then make it executable `chmod +x geckodriver`, also making the chromedriver executable should solve the Chrome permissions issue.

**Comment by jeremycjang** [comment]
Just use FireFox ESR, it’ll save you a lot of headache

**Comment by gustafsom** [comment]
Good afternoon, friends, all these days I was trying to get Instapy to run on Raspberry PI3. But without success.

I already tried the chromedriver. Initially it presented the error: `Message: 'chromedriver' executable may have wrong permission.
`
I ran the command: `chmod + x chromedriver`, but now it throws the error: `[errno 8] exec format error`

I installed geckodriver and Firefox-ers but the error occurs: `Element is not clickable at point (92, 92). Other element would receive the click: <ul class = "ueMLB"> </ ul>`

Trying to solve this problem, I downgrade and upgrade geckodriver, selenium, changed display resolution in `Instapy.py`, but to no avail.

I need some more help.
